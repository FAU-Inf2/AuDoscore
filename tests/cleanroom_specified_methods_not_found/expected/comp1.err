Exception in thread "main" java.lang.IllegalArgumentException: ERROR - ToTest.foo specified in @CompareInterface could not be found in cleanroom
	at tester.tools.CheckAnnotation.checkAnnotations(CheckAnnotation.java:76)
	at tester.tools.CheckAnnotation.main(CheckAnnotation.java:194)
make: *** [Makefile:51: compile-stage1] Fehler 1
