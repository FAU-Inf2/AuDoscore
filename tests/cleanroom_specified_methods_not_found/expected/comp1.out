make[1]: Entering directory '/home/sitowert/AuDoscore/tests/cleanroom_specified_methods_not_found/test.10356'
rm -f *.class */*.class
cp UnitTest.java UnitTest.java.orig ; javac -cp lib/junit.jar:lib/junitpoints.jar -proc:only -processor tools.bomacon.BonusMalusConverter UnitTest.java > UnitTest.java.tmp ; mv UnitTest.java.tmp UnitTest.java ; javac -cp lib/junit.jar:lib/junitpoints.jar -proc:only -processor FullQualifier UnitTest.java > UnitTest.java.tmp ; mv UnitTest.java.tmp UnitTest.java ; make -B UnitTest.class || ( mv UnitTest.java.orig UnitTest.java; /bin/false; ) ; mv UnitTest.java.orig UnitTest.java ;
make[2]: Entering directory '/home/sitowert/AuDoscore/tests/cleanroom_specified_methods_not_found/test.10356'
javac -cp lib/json-simple-1.1.1.jar:lib/junit.jar:lib/junitpoints.jar:. UnitTest.java ToTest.java 
make[2]: Leaving directory '/home/sitowert/AuDoscore/tests/cleanroom_specified_methods_not_found/test.10356'
javac cleanroom/*
java -cp lib/junit.jar:lib/junitpoints.jar:. CheckAnnotation UnitTest
Makefile:65: recipe for target 'compile-stage1' failed
make[1]: Leaving directory '/home/sitowert/AuDoscore/tests/cleanroom_specified_methods_not_found/test.10356'
